### ISSUE TITLE
Raspberry Pi install fails on cffi wheel due to missing dependencies

### ISSUE THREAD

**Issue by gnarlygeek** [dotnet/interactive#724]
### Describe the bug
When following the instructions here: 
        **https://github.com/dotnet/interactive/blob/main/docs/small-factor-devices.md** 
**The install on Raspberry PI 4 fails on the cffi wheel install due to a missing dependency.** 

The resolution is described in this StackOverflow item: https://stackoverflow.com/questions/22073516/failed-to-install-python-cryptography-package-with-pip-and-setup-py

The tl;dr is to run the following command and rerun the curl command: 
        **sudo apt-get install build-essential libssl-dev libffi-dev python-dev**

### Please complete the following:

- OS
    - [ ] Windows 10
    - [ ] macOS
    - [x ] Linux (Please specify distro) **"Raspbian GNU/Linux 10 (buster)"**
    - [ ] iOS
    - [ ] Android
 - Browser - N/A
 - Frontend N/A

**Comment by colombod** [comment]
@gnarlygeek is this issue resolved for you?

**Comment by gnarlygeek** [comment]
Hello Diego,

Yes it is. Sorry, I thought I had already replied. The script now works and the web page reference is corrected.

Thanks.

**Comment by colombod** [comment]
Closing this one then
